cp-ksql-server/include/etc/confluent/docker/configure

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o nounset \
+    -o errexit \
+    -o verbose \
+    -o xtrace
+
+dub ensure KSQL_BOOTSTRAP_SERVERS
+
+dub path /etc/"${COMPONENT}"/ writable
+
+if [[ -n "${KSQL_JMX_OPTS-}" ]]
+then
+  if [[ ! $KSQL_JMX_OPTS == *"com.sun.management.jmxremote.rmi.port"*  ]]
+  then
+    echo "KSQL_OPTS should contain 'com.sun.management.jmxremote.rmi.port' property. It is required for accessing the JMX metrics externally."
```

Comment:
- Typo: "KSQL_OPTS should contain" -> "KSQL_OPTS should contain the"
- We should perhaps also add a note to remind users to port-map the RMI port, otherwise it is not accessible from outside the container.  This should be obvious, but I think many people will still forget about that initially...